README.md

```diff
@@ -109,39 +109,9 @@ IRB `v1.13.0` and later versions allows users/libraries to extend its functional
 
 For more information, please visit the [IRB Extension Guide](https://ruby.github.io/irb/EXTEND_IRB_md.html).
 
-## Development
```

Comment:
I don't see this section providing much useful info as the releasing guideline is out of date too (pushing `.gem` isn't needed anymore). So I decided to drop it without replacement atm.